recipes/clinker/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "clinker" %}
+{% set version = "1.31" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: "v{{ version }}.tar.gz"
+  url: https://github.com/Oshlack/Clinker/archive/v{{ version }}.tar.gz
+  md5: "5487f99af6b9499c87d4d945e9cd9790"
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - samtools
+    - bpipe
+    - star 2.5.3a.*
+    - python 2.7.*
```

Comment:
python should not be pinned like this. Please use the build/skip section for this like `skip: True  # [py3k]`